packages/contracts-bedrock/.envrc.example

```diff
@@ -1,9 +1,12 @@
 # RPC for the network to deploy to
 export ETH_RPC_URL=
 
-# Sets the deployer's key to match the first default hardhat account
+# Set the deployer's key to match the first default hardhat account
 export PRIVATE_KEY=ac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80
 
+# Set IMPL_SALT to a random bytes32 hex string
+export IMPL_SALT=0xdb6d...random.bytes32...5303
```

Comment:
Would leave this commented out and note that this is an optional value with a default in the deploy script.

Reply:
For further context, see https://github.com/ethereum-optimism/optimism/blob/0b492be02342a7d32c09f114a3a976f961894511/packages/contracts-bedrock/scripts/Deploy.s.sol#L100

Reply:
Hi @clabby @refcell ,

Deploying with "ethers phoenix" salt is resulting in an error in getting-started deployment because it's trying to redeploy into existing contract addresses. Same as using the same salt across projects.

<img width="985" alt="Screenshot 2023-09-29 012041" src="https://github.com/ethereum-optimism/optimism/assets/4103490/d15703da-14bd-42eb-a56e-14a562217a22">

I think it should default to a random string instead of "ethers phoenix"?

Reply:
We can temporarily set it to a random value, we are in the process of migrating to a setup where all superchains use the same implementations, so the guide will need to be updated

Reply:
Just seen that it's keccak256(bytes(...)) so IMPL_SALT can be any string not just bytes32

Reply:
I think using a combination of finalSystemOwner and l1StartingBlockTag as salt.

Using random salt will make the address change every time we run the deployment script which is not good and may cause more bugs.

Reply:
Currently set IMPL_SALT default to `keccak256(abi.encode(cfg.finalSystemOwner(), cfg.l1StartingBlockTag(), cfg.l2ChainID()))` and have set IMPL_SALT requirement to optional.